kit/src/main/resources/com/oracle/javafx/scenebuilder/kit/i18n/SceneBuilderKit.properties

```diff
@@ -71,6 +71,9 @@ label.qualifier.vertical = (vertical)
 log.info.explore.end = End exploring {0}
 log.info.explore.folder = Start exploring FOLDER {0}
 log.info.explore.jar = Start exploring JAR {0}
+log.info.explore.jar.results = Results of exploring JAR {0}
+log.info.explore.folder.results = Results of exploring FOLDER {0}
```

Comment:
This property is not in use and can be removed.

Reply:
Fixed